app/templates/eventView.html

```diff
@@ -34,16 +32,64 @@ <h3>{{eventData["program"].programName}} Event</h3>
           </ul>
          </div> <br />
     {% endif %}
+    <!--
     {% if isPastEvent %}
-        <p class="heading-size"><b>{{eventData.name}}</b> took place on <b>{{eventData.startDate}}</b> from <b>{{eventData.timeStart}}</b> to <b>{{eventData.timeEnd}}</b> and was located in <b>{{eventData.location}}</b>.</p>
-    {% else %}
-        <p class="heading-size"><b>{{eventData.name}}</b> will take place on <b>{{eventData.startDate}}</b> from <b>{{eventData.timeStart}}</b> to <b>{{eventData.timeEnd}}</b> and will be located in <b>{{eventData.location}}</b>.</p>
+        <p class="heading-size"><b>{{eventData.name}}</b> took place on <b>{{eventData.startDate}}</b> </p>
     {% endif %}
+    -->
+    <div class="container-fluid">
             <div class="row">
                 <div class="col-md-6">
+                    <img src="{{image}}" class="img-fluid">
+                    <!-- Image should be about 1280px x 836px (Width x Height)-->
+                </div>
+                <div class="col-md-6">
+                    <br>
+                    <div>   
+                        {% set event_title = eventData.name %}
+                        <h3 id="title">{{event_title}}</h3>
+                    </div>
+                    <br>
+                    <div style="display: flex; align-items: top;">
+                        <i class="bi bi-calendar" style="margin-right: 10px;"></i>
+                        {% set event_date = eventData.startDate %}
+                        <div>
+                            <h5 id="date" style="margin: 0;"> Date:</h5>
+                            <p style="margin: 0;">{{event_date}}</p>
+                        </div>    
+                    </div>
+                    <br>
+                    <div style="display: flex; align-items: top;">
+                        <i class="bi bi-clock" style="margin-right: 10px;"></i>
+                        {% set event_time = eventData.timeStart %}
+                        <div>
+                            <h5 id="time" style="margin: 0;">Time:</h5>
+                            <p style="margin: 0;">{{eventData.timeStart}} - {{eventData.timeEnd}}</p>
+                        </div>    
+                    </div>
+                    <br>
+                    <div style="display: flex; align-items: top;">
```

Comment:
Could we replace these style tags with bootstrap classes?

Reply:
You should be able to replace `display: flex` with `d-flex` and `align-items: top` with `align-top`. The new `d-flex` and `align-top` would be in a `class` instead of style since they are the Bootstrap version. I am pretty sure you can replace all of the content in `style` tags with an equivalent Bootstrap `class`.